### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by meta-sabir-mohammad**
11-07-2019

### CODE REVIEW

File: GET2019-E19-1219/src/Assignment1/Product.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package Assignment1;
+
```
Comment (ghost) [inline GET2019-E19-1219/src/Assignment1/Product.java:2]: Add comments for class to describe class functionality.

File: GET2019-E19-1219/src/Assignment1/Product.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		this.ProductId = ProductId;
+		this.ProductName = ProductName;
+		this.ProductPrice = ProductPrice;
+	}
+
+	public int getProductQuantity() {
+		return ProductQuantity;
+	}
+
+	public void setProductQuantity(int productQuantity) {
```
Comment (ghost) [inline GET2019-E19-1219/src/Assignment1/Product.java:33]: Mention comments for method. do for other methods also.

File: GET2019-E19-1219/src/Assignment1/ShoppingCart.java [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class ShoppingCart {
+
+	/**
+	 * AllProductList store product available to purchase
+	 */
+	private ArrayList<Product> AllProductList = new ArrayList<Product>();
```
Comment (ghost) [inline GET2019-E19-1219/src/Assignment1/ShoppingCart.java:12]: Try to make class generic and use modular approach. Try to keep class specific data like all product list not required here.

File: GET2019-E19-1219/src/Assignment1/ShoppingCart.java [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+	/**
+	 * Cart store product in user shopping cart
+	 */
+	private HashMap<String, Product> Cart = new HashMap<String, Product>();
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		ShoppingCart shcart = new ShoppingCart();
+
+		Scanner UserInput = new Scanner(System.in);
```
Comment (ghost) [inline GET2019-E19-1219/src/Assignment1/ShoppingCart.java:23]: Try to follow naming conventions.

File: GET2019-E19-1219/src/Assignment1/ShoppingCart.java [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+
+	/**
+	 * 
+	 * @param product
+	 *            this contain detail about product
+	 * 
+	 */
+
+	private void addProduct(Product product) {
+
```
Comment (ghost) [inline GET2019-E19-1219/src/Assignment1/ShoppingCart.java:112]: Use exception handling whenever it required.

File: GET2019-E19-1219/src/Assignment1/Product.java (hunk 1 above)
Comment (meta-sabir-mohammad) [reply GET2019-E19-1219/src/Assignment1/Product.java:2]: ok sir i will make sure to add comments whenever it is required

File: GET2019-E19-1219/src/Assignment1/Product.java (hunk 2 above)
Comment (meta-sabir-mohammad) [reply GET2019-E19-1219/src/Assignment1/Product.java:33]: ok sir from now on i will add comments for methods

File: GET2019-E19-1219/src/Assignment1/ShoppingCart.java (hunk 3 above)
Comment (meta-sabir-mohammad) [reply GET2019-E19-1219/src/Assignment1/ShoppingCart.java:12]: ok sir in my upcoming assignment i will take modular approach and keep class specific data in there respective class

File: GET2019-E19-1219/src/Assignment1/ShoppingCart.java (hunk 4 above)
Comment (meta-sabir-mohammad) [reply GET2019-E19-1219/src/Assignment1/ShoppingCart.java:23]: ok sir

File: GET2019-E19-1219/src/Assignment1/ShoppingCart.java (hunk 5 above)
Comment (meta-sabir-mohammad) [reply GET2019-E19-1219/src/Assignment1/ShoppingCart.java:112]: ok sir from now on i will use exception handling whenever required

**Pull request closed without merging**